protocols/dcutr/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # 0.9.0 [unreleased]
 
+- Remove no longer required `ActionBuilder` from `Behaviour` implementation. See [PR 3304].
```

Comment:
I suggest not referencing this, given that it is not a user-facing change. Agreed?
```suggestion
```

Reply:
Thanks Max, yeah I was not so sure either. Updated!